docs/root/configuration/access_log.rst

```diff
@@ -164,6 +164,23 @@ The following command operators are supported:
   TCP
     Not implemented ("-").
 
+%DYNAMIC_METADATA(NAMESPACE:KEY*):Z%
+  :ref:`Dynamic Metadata <envoy_api_msg_core.Metadata>` info,
+  where NAMESPACE is the the filter namespace used when setting the metadata, KEY is an optional
+  lookup up key in the namespace with the option of specifying nested keys separated by ':',
+  and Z is an optional parameter denoting string truncation up to Z characters long. Dynamic Metadata
+  can be set by filters using the RequestInfo api: *setDynamicMetadata*. The data will be logged
```

Comment:
Nit: s/api/API/

Add for `RequestInfo` with a `:repo` link, same as well for `setDynamicMetadata`.

Reply:
Added RequestInfo link. Will add a link to setDynamicMetadata once the code is merged.